micromamba/tests/test_env.py

```diff
@@ -72,16 +72,22 @@ def export_env():
 
 @pytest.mark.parametrize("channel_subdir_flag", [None, "--channel-subdir"])
 @pytest.mark.parametrize("md5_flag", [None, "--md5", "--no-md5"])
-@pytest.mark.parametrize("explicit_flag", [None, "--explicit"])
-def test_env_export(export_env, explicit_flag, md5_flag, channel_subdir_flag):
-    flags = filter(None, [explicit_flag, md5_flag, channel_subdir_flag])
+@pytest.mark.parametrize(
+    "explicit_flag, json_flag", [(None, None), ("--explicit", None), (None, "--json")]
+)
```

Comment:
Note to reviewers: those two flags are incompatible, I think we need to properly handle the case when both are provided.